### PR TITLE
Laravel v9 support

### PR DESCRIPTION
**Pull request by promatik**
After https://github.com/barryvdh/elfinder-flysystem-driver/pull/83, Barry did it 🙌
https://github.com/barryvdh/laravel-elfinder/releases/tag/v0.5.0

Backpack can now fully support Laravel v9

### CODE REVIEW

**Comment by tabacitu** [comment]
Awesome! @pxpm please double-check?

**Comment by tabacitu** [comment]
I see they [dropped support for Laravel 8 and lower](https://github.com/barryvdh/laravel-elfinder/compare/v0.4.7...v0.5.0) in a minor version (0.5), so I think it's ok for FileManager to do the same. I don't think we'll be pushing any updates for L8 and lower specifically in this repo, ever.

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "keywords": ["Laravel", "FileManager"],
     "require": {
         "backpack/crud": "^4.1|^5.0",
```
Comment (promatik) [inline composer.json:15]:
```suggestion
        "backpack/crud": "^5.0",
```

Comment (tabacitu) [reply composer.json]: Agreed.

**Comment by tabacitu** [comment]
You know what? Let's launch this as a separate version, that only supports Backpack v5. It'll be easier that way. 

@pxpm if you test and give this your thumbs up, I can merge, tag, launch and we'll have ALL Backpack repos supporting L9! 💪

Thanks guys!!!

**Review by pxpm — APPROVED** [review]
image

image

Tested: Standalone, browse_multiple, browse and ckeditor.

**Comment by tabacitu** [comment]
Awesoooomee!!! Thanks guys! Merging and tagging now.

**Pull request merged**